### PR TITLE
Add bulletproofs to 1.7 release.yaml

### PR DESCRIPTION
**Pull request by mstraka100**
Adds bulletproofs natives feature flag to 1.7 release.yaml

### CODE REVIEW

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -52,3 +52,13 @@ proposals:
               sender_aware_v2: 32
             block_gas_limit: 35000
             transaction_deduper_type: txn_hash_and_authenticator_v1
+   - name: enable_bulletproofs
+     metadata:
+       title: "Enable Bulletproofs Module"
```
Comment (sherry-x) [inline aptos-move/aptos-release-builder/data/release.yaml:57]: add AIP number?

**Review by sherry-x — APPROVED** [review]

**Review by movekevin — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 2]
```diff
@@ -52,3 +52,13 @@ proposals:
               sender_aware_v2: 32
             block_gas_limit: 35000
             transaction_deduper_type: txn_hash_and_authenticator_v1
+   - name: enable_bulletproofs
```
Comment (sherry-x) [inline aptos-move/aptos-release-builder/data/release.yaml:55]: YAML is horrible, I think you have extra space on this line

Comment (mstraka100) [reply aptos-move/aptos-release-builder/data/release.yaml]: Correct. I think it is now fixed

**Comment by sherry-x** [comment]
I'm merging your change into my PR so we don't have to fight each other https://github.com/aptos-labs/aptos-core/pull/9845

**Comment by sherry-x** [comment]
closing cause I merged the change in another PR, thanks Michael!

**Pull request closed without merging**